Sources/Parser.swift

```diff
@@ -67,15 +67,24 @@ public class TokenParser {
 
   public func nextToken() -> Token? {
     if tokens.count > 0 {
+        #if !swift(>=3.0)
```

Comment:
The indentation style here is all over the place.

<img width="344" alt="screen shot 2016-03-31 at 17 56 24" src="https://cloud.githubusercontent.com/assets/44164/14182041/ebfd4984-f769-11e5-95c4-6251c77a0e07.png">